### ISSUE TITLE
Player Decreasing / Ageing

### ISSUE THREAD

**Issue by Elketro** [xAranaktu/FIFA-18---Career-Mode-Cheat-Table#4]
Could you add a script that would make players decrease slower i.e. later in their age, for example to start decrease in their 33s? 

Would be great if it was somewhat customizable but a flat value will still be huge, I just can't understand why players like Bale and Alexis Sanchez are decreasing in just 2nd season of the career

**Comment by PlaceboPRS** [comment]
Yes I'd love something like this, maybe it can be a configurable age so those of us who don't want any ageing can set it to 45 or something? I'm manually "repairing" my players once a month as they're starting to decline at 29!

**Comment by xAranaktu** [comment]
Was it possible to change that in older FIFA versions by editing .ini file? If yes then just let me know which file to edit. Otherwise, I highly doubt I'll add it, you need to edit player's age manually.

**Comment by Elketro** [comment]
I don't know about older FIFA's but the file responsible for this is playergrowth.ini

**Comment by xAranaktu** [comment]
There are too many variables, I don't have time to test it.
You can try to edit playergrowth.ini by yourself with this tool:
https://github.com/xAranaktu/FIFA-18---iniToCT

**Comment by Elketro** [comment]
I've already tried but can't seem to find the right ones, do you have any ideas which ones are worth checking? I've tested AGE_(1-10) and RATIO_(1-10) in all CURVEs but I haven't noticed any changes yet.

**Comment by xAranaktu** [comment]
Nie mam pojecia. Z tego co widze to kazda krzywa sie przypisana do innych pozycji, wiec moze wystarczy skopiowac wartosci z [CURVE_1], bo pewnie bramkarze traca umiejetnosci najwolniej.

**Comment by Elketro** [comment]
Ciekawy pomysł, ale krzywa odpowiada także za rozwój, więc każdy będzie rozwijał się w tempie bramkarzy, również jeśli chodzi o balans pomiędzy mentalnością, fizycznością, a techniką. Przyjrzę się jeszcze późniejszym stadium rozwoju u różnych pozycji, jakbyś na coś wpadł to pisz, przetestuję.

**Comment by Elketro** [comment]
Testowałem sporo różnych opcji, ale jedyne co dotychczas udało mi się znaleźć to sposób na zmianę "starzenia się". Aktualnie zawodnik traci głównie fizyczne staty i jest w stanie stracić w jeden sezon po 20 speeda, staminy itp. Znalazłem sposób by rozłożyć te straty na wszystkie rodzaje umiejętności, więc teraz straci 3 speeda, 3 podań, 3 agresji itp itd, co uważam, za poprawę systemu starzenia się. Nie sprawdzałem jeszcze tylko wartości ABOVE_CURVE i BELOW_CURVE, nie mam pojęcia o co w tym chodzi. Zaczynam nawet myśleć, że szukam jednak tego w niewłaściwym .ini, bo już nic innego nie mogę znaleźć, a przecież gdzieś to musi być. Będę jeszcze szukał sposobu na opóźnienie procesu, ale aktualnie nie mam pojęcia co testować.

ENGLISH: I haven't been able to figure out the way to delay the decline of the player, but I've found the way to change the ageing process (players now lose all their attributes evenly, not just the physical ones). Will try to continue my searches.

**Comment by Elketro** [comment]
Mógłbyś dodać playergrowth.ini do CT tak jak to zrobiłeś z transfer.ini ? Może to by pomogło w edycji krzywych.

**Comment by xAranaktu** [comment]
Przerzucenie calych plikow .ini do tabeli zajmuje strasznie duzo czasu, a efekt jest raczej odwrotny do zamierzonego. Wiec odkad FrostyTool dziala z FIFA 18 to nie widze sensu tego robic.
